app/views/shared/_history_notice.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if content_item.historically_political? %>
```

Comment:
I think this shared partial should also be in a separate commit, which is then followed by the generation of the locale keys in https://github.com/alphagov/frontend/pull/4573/commits/c08cd5b92c591f63772f35078862ba2687e23b57